<details><summary>Linter check found the following problems:</summary>
Traceback (most recent call last):
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/lint/__init__.py", line 823, in lint_file
    recipe = _recipe.Recipe.from_file(
        recipe_fname=str(meta_yaml),
    ...<2 lines>...
        renderer=RendererType.RUAMEL,
    )
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/percy/render/recipe.py", line 245, in from_file
    raise exc from exc
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/percy/render/recipe.py", line 236, in from_file
    recipe._load_from_string(text.read())
    ~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/percy/render/recipe.py", line 172, in _load_from_string
    self.render()
    ~~~~~~~~~~~^^
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/percy/render/recipe.py", line 297, in render
    self.meta = renderer_utils.render(self.recipe_file, self.dump(), self.selector_dict, self.renderer)
                ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/percy/render/_renderer.py", line 252, in render
    raise JinjaRenderFailure(recipe_file, message=exc.message, line=exc.lineno) from exc
percy.render.exceptions.JinjaRenderFailure: (PosixPath('/tmp/abs_0e7n746gjk/clone/recipe/meta.yaml'), "expected token 'end of statement block', got '{' (at line 42)")

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/run.py", line 208, in main
    return_code, report = execute_linter(
                          ~~~~~~~~~~~~~~^
        recipe=args.recipe,
        ^^^^^^^^^^^^^^^^^^^
    ...<6 lines>...
        verbose_flag=args.verbose,
        ^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/run.py", line 180, in execute_linter
    result = linter.lint(recipes, subdir, variant_config_files, exclusive_config_files, fix_flag)
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/lint/__init__.py", line 725, in lint
    msgs = self.lint_file(
        recipe_name,
    ...<3 lines>...
        fix=fix,
    )
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/lint/__init__.py", line 840, in lint_file
    recipe = _recipe.Recipe(recipe_name)
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/percy/render/recipe.py", line 109, in __init__
    self.recipe_dir = recipe_file.parent
                      ^^^^^^^^^^^^^^^^^^
AttributeError: 'str' object has no attribute 'parent'

ERROR conda.cli.main_run:execute(127): `conda run conda-lint /tmp/abs_0e7n746gjk/clone` failed. (See above for error)
</details>

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -27,7 +27,7 @@ requirements:
 
 {% set tests_to_skip = "" %}
 # AssertionError: 13970222226.416412 != 13970222226.413956 within 3 places (0.0024566650390625 difference)
-{% set tests_to_skip = '-k "not test_time"'' %}  # [aarch64]
+{% set tests_to_skip = '-k "not test_time"' %}  # [aarch64]
 
 test:
   source_files:

